!SESSION 2012-05-08 22:20:26.177 -----------------------------------------------
eclipse.buildId=M20120208-0800
java.version=1.7.0_03
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.reporting.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.reporting.product

!ENTRY org.eclipse.egit.core 4 0 2012-05-08 22:57:26.110
!MESSAGE An exception occurred during push on URI git@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: git@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:227)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:145)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:202)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.eclipse.jgit.errors.TransportException: git@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 4 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 11 more

!ENTRY org.eclipse.egit.core 4 0 2012-05-08 22:57:55.673
!MESSAGE An exception occurred during push on URI git@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: git@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:227)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:145)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:202)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.eclipse.jgit.errors.TransportException: git@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 4 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 11 more
!SESSION 2012-05-08 23:00:38.850 -----------------------------------------------
eclipse.buildId=M20120208-0800
java.version=1.7.0_03
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.reporting.product
Command-line arguments:  -data C:\Users\Alex Hepting\gruppe18 -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.reporting.product

!ENTRY org.eclipse.egit.core 4 0 2012-05-08 23:01:16.444
!MESSAGE Cache doesn't contain data for key: org.eclipse.ui.intro
!STACK 0
java.lang.RuntimeException: Cache doesn't contain data for key: org.eclipse.ui.intro
	at org.eclipse.egit.core.synchronize.GitSyncObjectCache.addMember(GitSyncObjectCache.java:83)
	at org.eclipse.egit.core.synchronize.GitSyncCache.loadDataFromGit(GitSyncCache.java:140)
	at org.eclipse.egit.core.synchronize.GitSyncCache.getAllData(GitSyncCache.java:81)
	at org.eclipse.egit.core.synchronize.GitSyncCache.getAllData(GitSyncCache.java:61)
	at org.eclipse.egit.core.synchronize.GitSyncCache.getAllData(GitSyncCache.java:50)
	at org.eclipse.egit.core.synchronize.GitResourceVariantTreeSubscriber.init(GitResourceVariantTreeSubscriber.java:84)
	at org.eclipse.egit.ui.internal.synchronize.GitModelSynchronize$2.run(GitModelSynchronize.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.equinox.security 1 0 2012-05-08 23:04:21.084
!MESSAGE New keyring password generated. 

!ENTRY org.eclipse.egit.core 4 0 2012-05-08 23:04:57.359
!MESSAGE An exception occurred during push on URI alhep100@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: alhep100@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:194)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:145)
	at org.eclipse.egit.ui.internal.push.SimpleConfigurePushDialog$17.run(SimpleConfigurePushDialog.java:617)
	at org.eclipse.jface.operation.ModalContext.runInCurrentThread(ModalContext.java:464)
	at org.eclipse.jface.operation.ModalContext.run(ModalContext.java:372)
	at org.eclipse.jface.dialogs.ProgressMonitorDialog.run(ProgressMonitorDialog.java:507)
	at org.eclipse.egit.ui.internal.push.SimpleConfigurePushDialog.buttonPressed(SimpleConfigurePushDialog.java:603)
	at org.eclipse.jface.dialogs.Dialog$2.widgetSelected(Dialog.java:624)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:240)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1053)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4165)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3754)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:825)
	at org.eclipse.jface.window.Window.open(Window.java:801)
	at org.eclipse.egit.ui.internal.push.PushResultDialog$2.run(PushResultDialog.java:94)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4140)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3757)
	at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2701)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2665)
	at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2499)
	at org.eclipse.ui.internal.Workbench$7.run(Workbench.java:679)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:668)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:123)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:344)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:622)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:577)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1410)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1386)
Caused by: org.eclipse.jgit.errors.TransportException: alhep100@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 41 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 48 more

!ENTRY org.eclipse.egit.core 4 0 2012-05-08 23:05:13.688
!MESSAGE An exception occurred during push on URI alhep100@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: alhep100@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:227)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:145)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:202)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.eclipse.jgit.errors.TransportException: alhep100@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 4 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 11 more
!SESSION 2012-05-10 08:08:26.370 -----------------------------------------------
eclipse.buildId=M20120208-0800
java.version=1.7.0_03
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Framework arguments:  -product org.eclipse.epp.package.reporting.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.reporting.product

!ENTRY org.eclipse.egit.core 4 0 2012-05-10 09:43:56.948
!MESSAGE An exception occurred during push on URI alhep100@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: alhep100@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:227)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:145)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:202)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.eclipse.jgit.errors.TransportException: alhep100@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 4 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 11 more

!ENTRY org.eclipse.egit.core 4 0 2012-05-10 09:44:38.278
!MESSAGE An exception occurred during push on URI alhep100@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: alhep100@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:227)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:145)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:202)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.eclipse.jgit.errors.TransportException: alhep100@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 4 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 11 more

!ENTRY org.eclipse.egit.core 4 0 2012-05-10 09:56:51.387
!MESSAGE An exception occurred during push on URI alhep100@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: alhep100@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:227)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:145)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:202)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.eclipse.jgit.errors.TransportException: alhep100@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 4 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 11 more

!ENTRY org.eclipse.egit.core 4 0 2012-05-10 09:58:20.387
!MESSAGE An exception occurred during push on URI git@github.com:propra12-orga/gruppe18.git: An internal Exception occurred during push: git@github.com:propra12-orga/gruppe18.git: session is down
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:156)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:194)
	at org.eclipse.egit.ui.internal.push.PushWizard$PushJob.run(PushWizard.java:295)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.eclipse.jgit.errors.TransportException: git@github.com:propra12-orga/gruppe18.git: session is down
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:154)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:118)
	at org.eclipse.jgit.transport.JschSession.exec(JschSession.java:91)
	at org.eclipse.jgit.transport.TransportGitSsh$SshPushConnection.<init>(TransportGitSsh.java:306)
	at org.eclipse.jgit.transport.TransportGitSsh.openPush(TransportGitSsh.java:152)
	at org.eclipse.jgit.transport.PushProcess.execute(PushProcess.java:130)
	at org.eclipse.jgit.transport.Transport.push(Transport.java:1120)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:152)
	... 3 more
Caused by: com.jcraft.jsch.JSchException: session is down
	at com.jcraft.jsch.Session.openChannel(Session.java:762)
	at org.eclipse.jgit.transport.JschSession$JschProcess.<init>(JschSession.java:147)
	... 10 more

!ENTRY org.eclipse.egit.ui 4 0 2012-05-10 09:58:20.427
!MESSAGE Can't connect to any repository: git@github.com:propra12-orga/gruppe18.git (An internal Exception occurred during push: git@github.com:propra12-orga/gruppe18.git: session is down)
